modules/core/src/lib/deck-renderer.ts

```diff
@@ -82,6 +82,9 @@ export default class DeckRenderer {
     }
 
     const outputBuffer = this.lastPostProcessEffect ? this.renderBuffers[0] : renderOpts.target;
+    if (this.lastPostProcessEffect) {
+      renderOpts.clearColor = [0, 0, 0, 0];
+    }
```

Comment:
If the intention here is to clear only when drawing to the canvas, might this also need to consider the `clearCanvas` option above rather than overriding it?

Reply:
That isn't the intention, adding this render option makes sure the ping pong buffers used in the post process effects get cleared when drawn to - this is separate to whether the canvas clearing